gazebo_msgs/CMakeLists.txt

```
@@ -2,6 +2,20 @@ cmake_minimum_required(VERSION 3.5)
 
 project(gazebo_msgs)
 
+if(NOT DEFINED IGNORE_GAZEBO_MSGS_WARNING)
+  if(DEFINED ENV{ROS_DISTRO})
+    set(ROS_DISTRO $ENV{ROS_DISTRO})
+    if(NOT (ROS_DISTRO STREQUAL "iron" OR
+            ROS_DISTRO STREQUAL "humble"))
+      message(WARNING "This gazebo_msgs package hosts messages designed initially for Gazebo Classic "
+        "which is not available on ROS 2 Jazzy which is unavailable on ROS 2 Jazzy. "
```

Is this what you meant?
```suggestion
        "which is not available on ROS 2 Jazzy which is unavailable on Ubuntu Noble. "
```